packages/prosemirror-lwdita/src/plugin.ts

```diff
@@ -93,6 +93,7 @@ export function shortcuts(schema: Schema) {
     'Ctrl-z': undo,
     'Ctrl-y': redo,
     'Ctrl-Shift-z': redo,
+    'Alt-n': insertImage(schema.nodes.image),
```

Comment:
@marmoure This key combination doesn't seem to work on my mac keyboard.

Reply:
This is still not working on a Mac keyboard: `alt + n `-> `~`

Reply:
@plutonik-a I was hoping this will show up in tests, but it didn't

Reply:
@marmoure What is the status here?

Reply:
I didn't manage to reproduce this can you run the test suit on your machine please.